android-core/src/main/java/com/mparticle/MParticleOptions.java

```diff
@@ -446,17 +457,33 @@ public Builder devicePerformanceMetricsDisabled(boolean disabled) {
         }
 
         /**
-         * By default, the SDK will collect <a href="http://developer.android.com/reference/android/provider/Settings.Secure.html#ANDROID_ID">Android Id</a> for the purpose
-         * of anonymous analytics. If you're not using an mParticle integration that consumes Android ID, the value will be sent to the mParticle
-         * servers and then immediately discarded. Use this API if you would like to additionally disable it from being collected entirely.
+         * By default, the SDK will NOT collect <a href="http://developer.android.com/reference/android/provider/Settings.Secure.html#ANDROID_ID">Android Id</a> for the purpose
+         * of anonymous analytics. If you're not using an mParticle integration that consumes Android ID and you would like to collect it, use this API to enable collection. The value will be sent to the mParticle
+         * servers and then immediately discarded.
          *
-         * @param disabled true to disable collection (false by default)
+         * @param disabled false to enable collection (true by default)
          *
          * @return the instance of the builder, for chaining calls
          */
         @NonNull
+        @Deprecated
         public Builder androidIdDisabled(boolean disabled) {
-            this.androidIdDisabled = disabled;
+            this.androidIdEnabled = !disabled;
+            return this;
+        }
+
+        /**
+         * By default, the SDK will NOT collect <a href="http://developer.android.com/reference/android/provider/Settings.Secure.html#ANDROID_ID">Android Id</a> for the purpose
+         * of anonymous analytics. If you're not using an mParticle integration that consumes Android ID and you would like to collect it, use this API to enable collection. The value will be sent to the mParticle
```

Comment:
this is tricky - it's not actually true that we'd discard it. i'd just kill this comment

Reply:
cool, I thought that sounded a little off. fixed now